docs/deployment/kubernetes.md

```diff
@@ -1,6 +1,33 @@
 ---
-title: Kubernetes
+title: Kubernetes with Helm
 sidebar_position: 2
 ---
 
-Native support for Kubernetes is not yet supported but is expected to land in Quickwit 0.4.
\ No newline at end of file
+[Helm](https://helm.sh) is a package manager for Kubernetes that allows you to configure, install, and upgrade containerized applications in a Kubernetes cluster. You can use the [official chart](https://github.com/quickwit-oss/helm-charts) for Quickwit to define your own configuration in a YAML file (`values.yaml`) and use Helm to deploy your applications in a version-controlled and reproducible way.
+
+# Install Quickwit using Helm
+
+1. Install the Helm CLI
+
+Select the appropriate method for your environment on Helm's [install documentation page](https://helm.sh/docs/intro/install/).
+
+2. Add the Quickwit Helm chart repository to Helm:
+
+    helm repo add quickwit https://helm.quickwit.io
+
+3. Update the repository
+
+    helm repo update
+
+4. Create and customize your configuration file `values.yaml`
```

Comment:
What seems to be useful here is that you can use `helm show values [CHART] [flags]` to get the default values file. At least, I found it useful when install the kube-prometheus and otel charts.